packages/engine/src/templates/postprocessing/DefaultPostProcessingSchema.ts

```diff
@@ -7,11 +7,34 @@ import { ToneMappingEffect } from '../../postprocessing/effects/ToneMappingEffec
 import { HueSaturationEffect } from '../../postprocessing/effects/HueSaturationEffect';
 import { DepthOfFieldEffect } from '../../postprocessing/effects/DepthOfFieldEffect';
 import { BrightnessContrastEffect } from '../../postprocessing/effects/BrightnessContrastEffect';
+import { Resizer } from '../../postprocessing/core/Resizer';
+import { OutlineEffect } from '../../postprocessing/effects/OutlineEffect';
+import { Engine } from '../../ecs/classes/Engine';
+
```

Comment:
remove this two line.
Engine is not used in this file. And there is no need in extra empty line.